mavproxy/source/docs/modules/firmware.rst

```diff
@@ -6,8 +6,7 @@ Firmware Management
 
     module load firmware
     
-The firmware module provides downloading and listing of available APM firmware. It 
-does not currently handle firmware flashing
+The firmware module provides downloading and listing of available APM firmware.

```

Comment:
let's use autopilot instead of apm...I need to globally change this in MAVProxy docs at some point

Reply:
Fixed